formal-spec/candidateproperties.tex

```diff
@@ -0,0 +1,153 @@
+\section{Candidate Properties}
+
+\subsection{Global Properties}
+
+\begin{property}[Software Update Proposal]\label{prop:submissions-for-all}
+Any stakeholder will always be able to propose a software update to be voted by the stakeholders' community and this will be recorded as an on-chain immutable event on the blockchain.
+\end{property}
+
+\begin{property}[Non Approved Software Updates]\label{prop:nonapproved-sus}
+A software update that is not approved by the stakeholders' community will never be applied,
```

Comment:
Don't we need to mention here a super majority?

About the classification:

- We can test that all proposals will be approved only if they have at least `tauV` percent of the stake.
- However, the proof that a given `tauV` is a sensible default is mathematical (and this is what you have in the paper, right?).